apps/activitypub/src/api/activitypub.ts

```diff
@@ -450,8 +450,10 @@ export class ActivityPubAPI {
         return this.getPaginatedPosts('.ghost/activitypub/v1/feed/reader', next);
     }
 
-    async getGlobalFeed(next?: string): Promise<PaginatedPostsResponse> {
-        return this.getPaginatedPosts('.ghost/activitypub/v1/feed/global', next);
+    async getGlobalFeed(next?: string, topic?: string): Promise<PaginatedPostsResponse> {
+        const topicPath = topic && topic !== 'all' ? topic : 'top';
```

Comment:
question: what's the purpose of that line of code? Wouldn't the `TOPICS` array in TopicFilter.tsx already pass the correct value?

Reply:
Yeah, this doesn't make sense. As we made it consistent by using `top` across everything, we wouldn't need this at all.